ocw-course-v2/ocw-studio.yaml

```
@@ -109,6 +110,68 @@ collections:
           filter_type: "equals"
           value: "Video"
 
+  - category: Content
+    name: link
+    label: Links
```

```suggestion
    name: external-resource
    label: External Resources
```

I don't care too much about the name, but we decided to call these "External Resources" because they're not just links. And they will behave like resources in every other way.